src/components/AddBook.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] [REQUIRED] Great job so far @Rimuru-Tempest-26 , however, Let's not disable Eslint because  ESLint helps enforce best practices and coding standards, which can improve the overall quality of the codebase. This leads to more maintainable, reliable, and scalable code.
Kindly fix the errors manually before requesting another review 💪🏽

Reply:
Fixed.

Reply:
- Well resolved 👍🏽